### PR TITLE
start on #377 for lim<-

### PR DESCRIPTION
**Pull request by jordansread**
set lims:

``` r
usrDef <- gsplot() %>% 
 points(x=1, y=2, side=c(3,2), legend.name="Points 1", cex=3, xlab='cat')
lim(usrDef,3)
[1] 1 1

lim(usrDef, 3)[1] <- 0
lim(usrDef,3)
[1] 0 1

lim(usrDef,3) <- c(0.25, NA)
lim(usrDef,3)
[1] 0.25 1.00

usrDef$side.3$usr.lim
[1]  TRUE FALSE
```

### CODE REVIEW

**Comment by jordansread** [comment]
``` r
usrDef <- gsplot() %>% 
    points(x=1, y=2, side=c(3,2), legend.name="Points 1", cex=3, xlab='cat')

lim(usrDef, side=3) <- c(0.2,NA)
usrDef <- points(usrDef, 5,2, side=c(3,2))
usrDef
```

image

**Comment by jordansread** [comment]
now this supports lists:

``` r
myplot <- gsplot() %>%
   points(1:3, 2:4) %>%
   lines(1:5, 5:1)
lim(myplot) <- list('side.1'=c(0.5,4))

lim(myplot)
$side.1
[1] 0.5 4.0

$side.2
[1] 1 5
```

File: README.md [hunk 1]
```diff
@@ -20,16 +20,52 @@ Overview
...
-| [Automatic legend](#legend)           | For each plot feature, a corresponding legend name can be specified. Eliminates the need to duplicate `par` arguments. |
-| [Automatic limits](#limits)           | Automatically regenerate limits of plot when new features are added.                                                   |
-| [Error bars](#error)                  | Apply error bars to points by specifying the upper and lower offsets.                                                  |
-| [Callouts](#callouts)                 | Add a line and label in one call.                                                                                      |
-| [Axes reversal](#reverse)             | Reverse the axis by specifying one argument to `axis()`.                                                               |
-| [Embedded functions](#embed)          | Add extra plot features within a points call (no duplication of x/y values)                                            |
-| [Compatibility with base](#base)      | Can start a plot using `gsplot`, and add base R features afterwards.                                                   |
+<table style="width:57%;">
```
Comment (ldecicco-USGS) [inline README.md:23]: Why the switch to html?

Comment (jiwalker-usgs) [inline README.md:23]: I think it is a knitr version difference, when I built this the other day it switched to markdown.

**Pull request closed without merging**